app/jobs/cache/browser_channels/responses_for_prefix.rb

```diff
@@ -46,7 +46,7 @@ def generate_brotli_encoded_channel_response(prefix:)
           wallet = PublishersPb::Wallet.new
           gemini_wallet = PublishersPb::GeminiWallet.new
           gemini_wallet.wallet_state = get_gemini_wallet_state(gemini_connection: site_banner_lookup.publisher.gemini_connection)
-          gemini_wallet.address = site_banner_lookup.publisher.gemini_connection.recipient_id
+          gemini_wallet.address = site_banner_lookup.channel.gemini_recipient_id
```

Comment:
Can we revert this for now and add it in later? That way there's no issues for users during tipping (I assume it'll be nil when users fetch this publisher's list)

Reply:
Per private PM, going to default this to the deposit ID on the connection since channels might not yet have them.